pages/views/__init__.py

```diff
@@ -37,3 +40,14 @@ def get(self, request, *args, **kwargs):
         app = get_current_account()
         kwargs.update({self.slug_url_kwarg: app})
         return super(AccountRedirectView, self).get(request, *args, **kwargs)
+
+class IndexView(TemplateView):
```

Comment:
Move this class into testsite/ instead of pages/ as it is not really part of the pages app, plus it is loading a template in testsite/ instead of pages/.